cockroachcloud/configure-cloud-org-sso.md

```diff
@@ -134,23 +134,42 @@ By default, members can access your {{ site.data.products.db }} organization fro
+SSO autoprovisioning allows members to sign up for an account without waiting for an invitation. By default, SSO autoprovisioning is disabled. When SSO autoprovisioning is disabled, a member must exist in the SSO provider and must be [invited by a console admin](console-access-management.html#console-admin) before they can create an account. When SSO autoprovisioning is enabled, no invitation is required.
 
-- A member's email address must exist in the SSO IdP.
-- A [console admin](console-access-management.html#console-admin) must have already invited the member to the {{ site.data.products.db }} organization.
+If you enable SSO autoprovisioning, Cockroach Labs recommends that you also limit the [allowed email domains](#allowed-email-domains) for the authentication method.
 
-Autoprovisioning allows members to access your organization without an invitation. Members are assigned the [Developer role](/docs/cockroachcloud/console-access-management.html#developer) by default. If you enable autoprovisioning, Cockroach Labs recommends that you also limit the [allowed email domains](#allowed-email-domains) for the authentication method.
+Members are assigned the [Developer role](/docs/cockroachcloud/console-access-management.html#developer) by default.
 
-To enable autoprovisioning for an SSO authentication method:
+If the member is removed from the SSO provider, they can no longer log in to {{ site.data.products.db }}, but their account is not automatically deprovisioned. If you require automatic deprovisioning or other centralized account automation features, refer to [SCIM Autoprovisioning](#scim-autoprovisioning). Cockroach Labs does not recommend enabling both SSO autoprovisioning and SCIM autoprovisioning for the same authentication method.
+
+To enable SSO autoprovisioning for an SSO authentication method:
 
 1. Log in to [{{ site.data.products.db }} Console](https://cockroachlabs.cloud) as a [console admin](console-access-management.html#console-admin).
 1. Go to **Organization** > **Authentication**.
 1. Click the name of an authentication method.
 1. Click **Advanced Settings**.
 1. Next to **Autoprovisioning**, click **Enable**.
 
+### SCIM Autoprovisioning
+
+[System for Cross-Domain Identity Management SCIM](https://www.rfc-editor.org/rfc/rfc7644) autoprovisioning centralizes and automates provisioning and deprovisioning of {{ site.data.products.db }} organization users from your IdP.
+
+Rather than a self-service approach, SCIM autoprovisioning tasks are performed by IdP admins. Cockroach Labs does not recommend enabling both SSO autoprovisioning and SCIM autoprovisioning for the same authentication method.
+
+1. In {{ site.data.products.db }}, an admin creates a [service account](console-access-management.html#service-accounts) and grants it the `ADMIN` role.
+1. In your IdP, an admin creates a SCIM application and configures it to use the service account to authenticate to the {{ site.data.products.db }} API.
```

Comment:
Do you think it would be worth calling out our base scim url of `https://cockroachlabs.cloud/api/scim/v2` up here too or leave that for the details below. That isn't an Okta specific thing.